tests/e2e/specs/backend/site-editing-templates.test.js

```diff
@@ -155,6 +155,11 @@ describe( 'Store Editing Templates', () => {
 	} );
```

Comment:
Now that this `beforeAll()` is inside each inner clause, do you think we can remove it from the top?

Reply:
Yes, I will change that and disable the test. 👍

Reply:
> Do you think we should disable those test until we can update e2e-test-utils and get them pass?

> Yes, I will change that and disable the tests. 👍

Did we log this somewhere so we can remember to 'unskip' those tests again when we update `e2e-test-utils`?

Reply:
Yep, I created an issue this morning https://github.com/woocommerce/woocommerce-blocks/issues/7744